test/api-endpoints/sequelize/balance.routes.sequelize.spec.ts

```diff
@@ -0,0 +1,226 @@
+  cryptocurrency: cryptocurrency.name,
+};
+const balanceTesting2 = {
+  client: user1.name,
+  amount: randomNumber(9),
+  cryptocurrency: cryptocurrency.name,
+};
+
+const balanceWithInvalidName = {
+  client: randomString(10),
+  amount: randomNumber(9),
+  cryptocurrency: cryptocurrency.name,
+};
+
+const balanceWithInvalidCryptocurrency = {
+  client: user1.name,
+  amount: randomNumber(9),
+  cryptocurrency: randomNumber(9),
+};
+
+describe("balance endpoints", () => {
+  before("creates client and cryptocurrency", async () => {
+    await clientRepository.create(user1);
+    await cryptocurrencyRepository.create(cryptocurrency);
+  });
+
+  after("deletes client and cryptocurrency", async () => {
+    await clientRepository.remove(user1.name);
+    await cryptocurrencyRepository.remove(cryptocurrency.name);
+  });
+
+  describe("sequelize/balance/create/Carlos", () => {
+    it("should create a balance if not exists, and save it to database", (done) => {
+      chai
+        .request(app)
+        .post(`/sequelize/balance/create/${balanceTesting.client}`)
+        .set("content-type", "application/json")
+        .send(balanceTesting)
+        .end((err, res) => {
+          expect(res.status).to.equal(201);
+          done(err);
+        });
+    });
+    it("should not create a balance if it exists", (done) => {
+      chai
+        .request(app)
+        .post(`/sequelize/balance/create/${balanceTesting.client}`)
+        .set("content-type", "application/json")
+        .send(balanceTesting)
+        .end((err, res) => {
+          expect(res.status).to.equal(403);
+          done(err);
+        });
+    });
+
+    it("should not create a balance with invalid name", (done) => {
+      chai
+        .request(app)
+        .post(`/sequelize/balance/create/${balanceTesting.client}`)
+        .set("content-type", "application/json")
+        .send(balanceWithInvalidName)
+        .end((err, res) => {
+          expect(res.status).to.equal(500);
+          done(err);
+        });
+    });
+
+    it("should not create a balance with invalid cryptocurrency", (done) => {
+      chai
+        .request(app)
+        .post(`/sequelize/balance/create/${balanceTesting.client}`)
+        .set("content-type", "application/json")
+        .send(balanceWithInvalidCryptocurrency)
+        .end((err, res) => {
+          expect(res.status).to.equal(500);
+          done(err);
+        });
+    });
+  });
+
+  describe(`/sequelize/balance/get/${balanceTesting.client}/${balanceTesting.cryptocurrency}`, () => {
+    it("should show a balance if exists", (done) => {
+      chai
+        .request(app)
+        .get(
+          `/sequelize/balance/get/${balanceTesting.client}/${balanceTesting.cryptocurrency}`
+        )
+        .set("content-type", "application/json")
+        .send(balanceTesting)
+        .end((err, res) => {
+          const { client, amount, cryptocurrency } = res.body;
+          const requestedBalance = {
+            client: client,
```

Comment:
You can avoid putting the same name twice:
cliente: client, ---> client,
Same with the other attributtes